Gemfile

```diff
@@ -77,6 +77,8 @@ gem 'recipient_interceptor'
 # Use for DoS attacks
 gem 'rack-attack'
 
+gem 'webpacker'
```

Comment:
Can we version this? (By the way, i think the version 3.0 of webpacker just went out.
https://twitter.com/dhh/status/902982685945196544